### ISSUE TITLE
Add rule to flag comments with #:

### ISSUE THREAD

**Issue by Stigjb** [astral-sh/ruff#9114]
The E262/E265 rules checks if one space is used after inline/block comments. However, to be consistent with Pycodestyle, they will accept both `# ` and `#: ` as valid inline comments, which is inconsistent with the wording in PEP8. I believe I've heard that this quirk is because of some conventional usage of comments for documenting class attributes, but I'm unfortunately unable to find a reference.

I would like a linting rule that follows PEP 8 to the letter and allows only `# ` in block and/or inline comments.

Example:

```python
#bad block comment
#:bad block comment
#: bad block comment (but allowed by E265)
# good block comment

a = 1  #bad inline comment
b = 1  #:bad inline comment
c = 1  #: bad inline comment (but allowed by E262)
d = 1  # good inline comment
```
